src/tribler-core/tribler_core/modules/libtorrent/restapi/torrentinfo_endpoint.py

```diff
@@ -45,6 +45,14 @@ def setup_routes(self):
         """
 
         args = request.query
+
+        hops = None
+        if 'hops' in args:
+            try:
+                hops = int(args['hops'])
+            except ValueError:
+                return RESTResponse({"error": "wrong value of 'hops' parameter"}, status=HTTP_BAD_REQUEST)
```

Comment:
Can we please add the erroneous value to the response message? I noticed that sometimes we just throw errors without giving an exact specification of what's wrong, so in order to make our life easier when we have to debug something, I argue it's helpful to also include the wrong value.

Reply:
Agree